transformer.py

```diff
@@ -97,24 +101,32 @@ def generate_traits_list(traits: list) -> list:
 
     return trait_list
 
+
 def calculate_canopycover_masked(pxarray: np.ndarray) -> float:
     """Return greenness percentage of given numpy array of pixels.
-
-    Args:
-      pxarray (numpy array): rgb image
-
-    Return:
+    Arguments:
+      pxarray (numpy array): rgba image where alpha 255=data and alpha 0=NoData
+    Returns:
       (float): greenness percentage
     """
 
-    # For masked images, all nonzero pixels are considered canopy
-    nonzeros = np.count_nonzero(pxarray)
-    ratio = nonzeros/float(pxarray.size)
+    # If > 75% is NoData, return a -1 ccvalue for omission later
+    total_size = pxarray.shape[0] * pxarray.shape[1]
+    nodata = np.count_nonzero(pxarray[:, :, 2] == 0)
```

Comment:
This is an incorrect change made to match the data you have & not the data that it should be. Please change back here and below `[:, :, 3]`